mkdocs.yml

```diff
@@ -399,6 +399,7 @@ nav:
         - Importing historical user tracking data: personalization/importing_historical_user_tracking_data.md
         - Track with ibexa-tracker.js: personalization/tracking_with_ibexa-tracker.md
         - Attribute search in Elasticsearch: personalization/attribute_search_in_elasticsearch.md
+        - Personalization guide : personalization/personalization_brochure.md
```

Comment:
I would place it under "How Personalization works" or even replace it as this guide describes it better and we don't need the repetition.

Reply:
I would leave How personalization works anyway.